tasks/fetch_covid19_india_data_task.py

```diff
@@ -7,7 +7,7 @@
 class FetchCovid19IndiaDataTask(luigi.Task):
     """ Fetches all the data available trough the API without any filtering """
     url = luigi.Parameter(
-        default="https://raw.githubusercontent.com/covid19india/api/gh-pages/v4/data-all.json"
+        default="https://github.com/covid19india/api/raw/gh-pages/v4/min/data-all.min.json"
```

Comment:
I see the "total recovered" metrics when i open the new json. Thanks! 
But the project still has the issue you mentioned that the covid19india operation is winding down right?